operator/src/main/resources/application.properties

```diff
@@ -40,14 +40,26 @@ ingresscontroller.max-ingress-connections=108000
 # percentage of peak connections you actually need to meet
 ingresscontroller.peak-connection-percentage=100
 
-# Enables haproxy option contstats in the kas ingress so that stats are reported live rather than at connection close (RFE-3007 provide a proper mechanism to enable this option).
-ingresscontroller.ingress-container-command=/usr/bin/bash,-c,awk '{print $0} /^defaults$/ {print \"  option contstats\"}' < $TEMPLATE_FILE > /tmp/haproxy-config.template; exec /usr/bin/openshift-router --v=2 --template /tmp/haproxy-config.template
-# Disconnect established connections after a haproxy reconfiguration (NE-879 should eliminate the need for this)
+ingresscontroller.router-verbosity=2
+# note that this variable is overridden using the downward API in the deployment.
+ingresscontroller.blueprint-namespace=kas-fleetshard-operator
+
+# customizes the router operators command line to enable specific options that can't currently be controller from the CR.
+ingresscontroller.ingress-container-command=/usr/bin/bash,-c,awk '{print $0} /^defaults$/ {print \"  option contstats\"}' < $TEMPLATE_FILE > /tmp/haproxy-config.template; exec /usr/bin/openshift-router --v=${ingresscontroller.router-verbosity} --template /tmp/haproxy-config.template --commit-interval=${ingresscontroller.commit-interval} --blueprint-route-pool-size=${ingresscontroller.blueprint-route-pool-size} --max-dynamic-servers=${ingresscontroller.max-dynamic-servers} --blueprint-route-namespace=${ingresscontroller.blueprint-namespace}
+
+# HA proxy dynamic config manager
+ingresscontroller.dynamic-config-manager=true
+# the following three variables are meaningful when the dynamic-config-manager=true
+ingresscontroller.commit-interval=9223372036854775807ns
+ingresscontroller.blueprint-route-pool-size=500
+ingresscontroller.max-dynamic-servers=5
```

Comment:
@k-wall , I saw it will still do config reload if I created a "6 brokers" cluster. The error is:
```
I0117 09:03:08.081835       1 router.go:833] template "msg"="router will reload as the ConfigManager could not dynamically replace endpoints for service"  "backendKey"="kafka-cf367j0d66v84eqivrgg:mykafka-kafka-bootstrap" "error"="no free dynamic server slots for backend kafka-cf367j0d66v84eqivrgg:mykafka-kafka-bootstrap, 1 endpoint(s) remaining" "service"="kafka-cf367j0d66v84eqivrgg/mykafka-kafka-external-bootstrap" "weight"=1
```

Had a quick look at the code, looks like it means we have to increase the `max-dynamic-servers`. If we deployed 6 brokers, we might need per-broker (6) + bootstrap server + kafka admin = 8? I'm not 100% sure. Will keep investigating

Reply:
Looking at the generated haproxy config, the only only backend we have with multiple servers is the bootstrap route:

```
backend be_tcp:kafka-xxxxxxxxxxx:test-instance-xxxxxxxxx-95b6-11-kafka-bootstrap
  balance leastconn

  hash-type consistent
  timeout check 5000ms
  server pod:test-instance-xxxxxxxxx-95b6-11-kafka-1:test-instance-xxxxxxxxx-95b6-11-kafka-external-bootstrap:tcp-external:[10.128.92.150:9094](http://10.128.92.150:9094/) [10.128.92.150:9094](http://10.128.92.150:9094/) weight 256 check inter 5000ms
  server pod:test-instance-xxxxxxxxx-95b6-11-kafka-2:test-instance-xxxxxxxxx-95b6-11-kafka-external-bootstrap:tcp-external:[10.129.92.157:9094](http://10.129.92.157:9094/) [10.129.92.157:9094](http://10.129.92.157:9094/) weight 256 check inter 5000ms
  server pod:test-instance-xxxxxxxxx-95b6-11-kafka-0:test-instance-xxxxxxxxx-95b6-11-kafka-external-bootstrap:tcp-external:[10.130.210.170:9094](http://10.130.210.170:9094/) [10.130.210.170:9094](http://10.130.210.170:9094/) weight 256 check inter 5000ms
```

so I think we'd need to set max-dynamic-servers to 6 to support 2SU.

You found:

> Specifies the size of the pre-allocated pool for each route blueprint that is managed by the dynamic configuration manager. This can be overriden on an individual route basis using the `router.openshift.io/pool-size` annotation on any blueprint route.

so we could potential add this information for the blueprint generated for the bootstrap case.  However we spoke about this and decided to keep things simple: raise the parameter globally.  When we support 3SU, we might want to revisit this.

Reply:
You're right! Changing `max-dynamic-servers=6` will make deploying 2SU cases without config reload. And the memory usage, compared with `max-dynamic-servers=5` case, it is increased form around 270Mi to 290Mi when deploying 6 brokers instance

Reply:
> However we spoke about this and decided to keep things simple: raise the parameter globally. When we support 3SU, we might want to revisit this.

Agree!

Reply:
I've bumped to max-dynamic-server to 6 to suit 2SU.